### ISSUE TITLE
how to get payload of the packet?

### ISSUE THREAD

**Issue by chuanic** [kohler/click#202]
Below is my packet:
    |   MAC header (14 bytes)   |   IP header (20 bytes)   |   TCP header  (20 bytes)   |   Options (12 bytes)   |   PAYLOAD   |

**Comment by bcronje** [comment]
I think you need to be a bit more specific as to what you want to do/achieve.

**Comment by ahenning** [comment]
yep, because Strip(66) is probably not what you want to do unless its a quick crude test

**Comment by chuanic** [comment]
sorry, i just want to compare one packet's payload with another packet's payload. But the means i have tried are not available which give me a message "segmentation fault".

**Comment by chuanic** [comment]
PS: how can i store a payload (40+ bytes) in a temp argument.

**Comment by chuanic** [comment]
Hi, guys!

Using Classifier can solve this problem!

**Comment by tbarbette** [comment]
So this is closed?
